src/ephemeris.c

```diff
@@ -88,8 +88,10 @@ static msg_t nav_msg_thread(void *arg)
       if (!es[ch->prn].healthy) {
         log_info("PRN %02d unhealthy\n", ch->prn+1);
       } else {
-        sbp_send_msg(SBP_MSG_EPHEMERIS, sizeof(ephemeris_t),
-                     (u8 *)&es[ch->prn]);
+        msg_ephemeris_t e;
+        pack_ephemeris(&e, (ephemeris_t *) &es[ch->prn]);
```

Comment:
I don't think it is required, I kind of like it for clarity so you don't have to go check the array of structures.